internal/clients/team4/iifo.go

```diff
@@ -0,0 +1,150 @@
+package team4
+
+import (
+	"math"
+
+	"github.com/SOMAS2020/SOMAS2020/internal/common/baseclient"
+	"github.com/SOMAS2020/SOMAS2020/internal/common/shared"
+)
+
+// MakeDisasterPrediction is called on each client for them to make a prediction about a disaster
+// Prediction includes location, magnitude, confidence etc
+// COMPULSORY, you need to implement this method
+func (c *client) MakeDisasterPrediction() shared.DisasterPredictionInfo {
+	// Use the sample mean of each field as our prediction
+	meanDisaster := getMeanDisaster(c.obs.pastDisastersList)
+
+	prediction := shared.DisasterPrediction{
+		CoordinateX: meanDisaster.CoordinateX,
+		CoordinateY: meanDisaster.CoordinateY,
+		Magnitude:   meanDisaster.Magnitude,
+		TimeLeft:    meanDisaster.Turn,
+	}
+
+	// Use (variance limit - mean(sample variance)), where the mean is taken over each field, as confidence
+	// Use a variance limit of 100 for now //TODO: tune this
+	varianceLimit := 100.0
```

Comment:
I hope you know what you're doing because I don't know what is varianceLimit and why it should be tuned

Reply:
I copied most of this part from baseclient and made changed some details. Personally I don't know how the confidence is used either, but I raised an issue on the mainmain repo about some potential problems I see. Will update this once I get a response